geolocator_android/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: geolocator_android
 description: Geolocation plugin for Flutter. This plugin provides the Android implementation for the geolocator.
-version: 3.0.0+4
+version: 3.0.0+5
```

Use the patch version here aswel:
```suggestion
version: 3.0.1
```